index.html

```diff
@@ -621,14 +619,11 @@ <h4>Signing Block Format</h4>
                 
             <section id="sec-signing-block-signature">
                 <h5>Signatures</h5>
-                <p>The signatures are stored as <strong>ID-value pairs</strong> with specific identifiers that represent the type of the signature. The supported signatures, and associated IDs, for RPK packages are the following:</p>
-                <ul>
-                    <li><code>0x01000101</code> Entire package signature, to guarantee the integrity of the MiniApp as a whole.</li>
-                    <li><code>0x01000201</code> File list signature</li>
-                </ul>
-
+                <p>The signatures are stored as <strong>ID-value pairs</strong> with specific identifiers that represent the types of the signature. The supported signatures, and associated IDs, for RPK packages are described in the following sub-clauses.</p>
+                
+                <p class="ednote" title="package-splitting"> There are cases where a MiniApp package is split into multiple sub-packages (by a distribution platform e.g. an app-store) for the user agent to download and execute it in a progressive fashion to improve the user experience. In such a case, additional signature types (thus IDs) would be needed to ensure the integrity of each sub-package as well as the whole. Details are ffs.</p>
```

Comment:
What does "ffs" mean here?

Reply:
Sorry, I update it to the full term - for further study.